test/salt/pillar/postgres.sls

```diff
@@ -1,7 +1,7 @@
 # Port to use for the cluster -- can be used to provide a non-standard port
 # NOTE: If already set in the minion config, that value takes priority
 
-{%- if not (grains.os_family == 'Debian' or grains.osfinger == 'Leap-15') %}
+{%- if not (grains.os_family == 'Debian' or salt['grains.get']('osfinger') == 'Leap-15')) %}
```

Comment:
CI fails because of too many parentheses, @noelmcloughlin :sweat_smile:

Reply:
there's always one!!